fs-extra/fs-extra.d.ts

```diff
@@ -171,6 +173,11 @@ declare module "fs-extra" {
 		flag?: string;
 	}
 
+	export interface MkdirOptions {
+		fs?: any;
```

Comment:
This is important for some unit tests which rely on fs-extra and mock-fs. So a merge would be super helpful :)